worlds/sc2/__init__.py

```diff
@@ -682,19 +677,17 @@ def pad_item_pool_with_filler(self: SC2World, num_items: int, pool: List[Item]):
         pool.append(item)
 
 
-def get_first_mission(mission_req_table: Dict[SC2Campaign, Dict[str, MissionInfo]]) -> SC2Mission:
-    # The first world should also be the starting world
-    campaigns = mission_req_table.keys()
-    lowest_id = min([campaign.id for campaign in campaigns])
-    first_campaign = [campaign for campaign in campaigns if campaign.id == lowest_id][0]
-    return list(mission_req_table[first_campaign].values())[0].mission
-
+def get_first_mission(world: SC2World, mission_order: SC2MissionOrder) -> SC2Mission:
+    # Pick an arbitrary lowest-difficulty starer mission
+    missions = mission_order.get_starting_missions()
+    missions = [(mission_order.mission_pools.get_modified_mission_difficulty(mission), mission) for mission in missions]
+    missions.sort()
```

Comment:
Throws `TypeError: '<' not supported between instances of 'SC2Mission' and 'SC2Mission'`. You need to use key:

```python
 missions.sort(key=lambda mission: mission.id)
```